ch_tools/monrun_checks/ch_orphaned_objects.py

```diff
@@ -48,6 +51,9 @@ def orphaned_objects_command(
     if not ClickhouseConfig.load().storage_configuration.has_disk("object_storage"):
         return Result(OK, "Disabled")
 
+    if get_uptime(ctx) == timedelta():
```

Comment:
Suggest to have some threshold on min uptime as in the backup check. It should be big enough (something like 1 day) because `ch-monitoring orphaned-objects` just report status from the state file and actual search of orphan objects is performed outside of this command.

Reply:
I agree, thank you, I corrected it